But in release config I see the same as you... Worrying that we have different warning flags for release...

```
mike@mike-desktop:~/dev/dronin2$ make all_clean
[ ! -d "/home/mike/dev/dronin2/build" ] || rm -rf "/home/mike/dev/dronin2/build"
mike@mike-desktop:~/dev/dronin2$ QT_SPEC=linux-g++6 GCS_BUILD_CONF=release make -j8 gcs 1>/dev/null
Project MESSAGE: QMAKE_CXX = g++-6
Project MESSAGE: QMAKE_CC = gcc-6
/home/mike/dev/dronin2/ground/gcs/src/libs/tlmapcontrol/mapwidget/mapline.cpp: In member function ‘virtual void mapcontrol::MapLine::paint(QPainter*, const QStyleOptionGraphicsItem*, QWidget*)’:
/home/mike/dev/dronin2/ground/gcs/src/libs/tlmapcontrol/mapwidget/mapline.cpp:94:5: error: this ‘if’ clause does not guard... [-Werror=misleading-indentation]
     if (line().dy() >= 0)
     ^~
/home/mike/dev/dronin2/ground/gcs/src/libs/tlmapcontrol/mapwidget/mapline.cpp:97:9: note: ...this statement, but the latter is misleadingly indented as if it is guarded by the ‘if’
         QPointF arrowP1 = line().pointAt(0.5) + QPointF(sin(angle + M_PI / 3) * arrowSize,
         ^~~~~~~
cc1plus: all warnings being treated as errors
make[4]: *** [.obj/release-shared/mapline.o] Error 1
make[4]: *** Waiting for unfinished jobs....
make[3]: *** [sub-tlmapcontrol-make_first-ordered] Error 2
make[2]: *** [sub-libs-make_first-ordered] Error 2
make[1]: *** [sub-src-make_first-ordered] Error 2
make: *** [gcs] Error 2
```

--- a/ground/gcs/src/libs/quazip/zip.c
+++ b/ground/gcs/src/libs/quazip/zip.c
@@ -527,13 +527,14 @@ local ZPOS64_T zip64local_SearchCentralDir(const zlib_filefunc64_32_def* pzlib_f
     if (ZREAD64(*pzlib_filefunc_def,filestream,buf,uReadSize)!=uReadSize)
       break;
 
-    for (i=(int)uReadSize-3; (i--)>0;)
-      if (((*(buf+i))==0x50) && ((*(buf+i+1))==0x4b) &&
-        ((*(buf+i+2))==0x05) && ((*(buf+i+3))==0x06))
-      {
-        uPosFound = uReadPos+i;
-        break;
-      }
+    for (i=(int)uReadSize-3; (i--)>0;) {
+        if (((*(buf+i))==0x50) && ((*(buf+i+1))==0x4b) &&
+            ((*(buf+i+2))==0x05) && ((*(buf+i+3))==0x06))
+        {
+            uPosFound = uReadPos+i;
+            break;
+        }
+    }
 
       if (uPosFound!=0)
         break;

--- a/ground/gcs/src/libs/tlmapcontrol/mapwidget/mapline.cpp
+++ b/ground/gcs/src/libs/tlmapcontrol/mapwidget/mapline.cpp
@@ -94,23 +94,22 @@ void MapLine::paint(QPainter *painter, const QStyleOptionGraphicsItem *option, Q
     if (line().dy() >= 0)
         angle = (M_PI * 2) - angle;
 
-        QPointF arrowP1 = line().pointAt(0.5) + QPointF(sin(angle + M_PI / 3) * arrowSize,
-                                        cos(angle + M_PI / 3) * arrowSize);
-        QPointF arrowP2 = line().pointAt(0.5) + QPointF(sin(angle + M_PI - M_PI / 3) * arrowSize,
-                                        cos(angle + M_PI - M_PI / 3) * arrowSize);
-        arrowHead.clear();
-        arrowHead << line().pointAt(0.5) << arrowP1 << arrowP2;
+    QPointF arrowP1 = line().pointAt(0.5) + QPointF(sin(angle + M_PI / 3) * arrowSize,
+                                                    cos(angle + M_PI / 3) * arrowSize);
+    QPointF arrowP2 = line().pointAt(0.5) + QPointF(sin(angle + M_PI - M_PI / 3) * arrowSize,
+                                                    cos(angle + M_PI - M_PI / 3) * arrowSize);
+    arrowHead.clear();
+    arrowHead << line().pointAt(0.5) << arrowP1 << arrowP2;
 
-        painter->drawPolygon(arrowHead);
-        if(myColor==Qt::red)
-            myPen.setWidth(3);
-        else if(myColor==Qt::yellow)
-            myPen.setWidth(2);
-        else if(myColor==Qt::green)
-            myPen.setWidth(1);
-        painter->setPen(myPen);
-        painter->drawLine(line());
-
+    painter->drawPolygon(arrowHead);
+    if(myColor==Qt::red)
+        myPen.setWidth(3);
+    else if(myColor==Qt::yellow)
+        myPen.setWidth(2);
+    else if(myColor==Qt::green)
+        myPen.setWidth(1);
+    painter->setPen(myPen);
+    painter->drawLine(line());
 }
 
 void MapLine::refreshLocations()

--- a/ground/gcs/src/plugins/coreplugin/manhattanstyle.cpp
+++ b/ground/gcs/src/plugins/coreplugin/manhattanstyle.cpp
@@ -61,7 +61,7 @@ bool styleEnabled(const QWidget *widget)
     while (p) {
         if (p->property("_q_custom_style_disabled").toBool())
             return false;
-            p = p->parentWidget();
+        p = p->parentWidget();
     }
     return true;
 }

--- a/ground/gcs/src/plugins/opmap/opmapgadgetwidget.cpp
+++ b/ground/gcs/src/plugins/opmap/opmapgadgetwidget.cpp
@@ -740,7 +740,7 @@ void OPMapGadgetWidget::zoomChanged(double zoomt, double zoom, double zoomd)
     if (m_widget->horizontalSliderZoom->value() != i_zoom)
         m_widget->horizontalSliderZoom->setValue(i_zoom);	// set the GUI zoom slider position
 
-	int index0_zoom = i_zoom - m_min_zoom;			// zoom level starting at index level '0'
+    int index0_zoom = i_zoom - m_min_zoom;			// zoom level starting at index level '0'
     if (index0_zoom < zoomAct.count())
         zoomAct.at(index0_zoom)->setChecked(true);		// set the right-click context menu zoom level
 }

